src/woo_publications/publications/admin.py

```diff
@@ -223,6 +225,20 @@ def save_model(
         is_published = new_status == PublicationStatusOptions.published
         is_revoked = new_status == PublicationStatusOptions.revoked
 
+        if not change:
+            retention_ic = get_retention_informatie_category(
+                form.cleaned_data["informatie_categorieen"]
+            )
+
+            if retention_ic:
```

Comment:
Could change to an assert because this should always have a record but this seemed saver?